src/sentry/features/manager.py

```diff
@@ -90,6 +97,11 @@ def has(self, name, *args, **kwargs):
         if rv is not None:
             return rv
 
+        if self._entity_handler:
```

Comment:
Should the docstring for this method be updated to reflect that the entity handler will also be checked?